Registry/Registry.EM_COMMON

```diff
@@ -2046,6 +2047,8 @@ rconfig   integer alloc_qndropsource      derived               1             0
 rconfig   integer   num_moves       namelist,domains    1                0
 rconfig   integer   ts_buf_size     namelist,domains    1                200          -       "ts_buf_size"   "Size of time series buffer"
 rconfig   integer   max_ts_locs     namelist,domains    1                5            -       "max_ts_locs"   "Maximum number of time series locations"
+rconfig   logical   tslist_ij       namelist,domains    1              .false.        rh      "tslist_ij"    "Use i,j locations in tslist"   ""
+rconfig   logical   tslist_interp   namelist,domains    1              .false.        rh      "tslist_interp"    "interpolate u and v to cell centers"   ""
```

Comment:
`tslist_interp` seems like an overly general name for an option that specifically controls whether the horizontal winds are interpolated from their C-grid staggering to an A-grid. Perhaps this option could be called something like `tslist_destagger_winds` or something similar?